src/main/java/com/github/igotyou/FactoryMod/listeners/CompactItemListener.java

```diff
@@ -49,6 +50,18 @@ public void craftingEvent(CraftItemEvent e) {
 		}
 	}
 
+	/**
+	 * Prevents players from eating compacted items
+	 */
+	@EventHandler
+	public void itemConsumeEvent(PlayerItemConsumeEvent e) {
+		if (isCompacted(e.getItem())) {
+			e.setCancelled(true);
+			e.getPlayer().sendMessage("You can not eat compacted food");
```

Comment:
Make these msgs red here and the other two events pls

Reply:
Done #